front-office/Your-vouchers.md

```diff
@@ -0,0 +1,31 @@
+## Rules of the displayed vouchers
+
+A voucher is displayed if it follows specific restrictions:
+
+On the table below are listed all the different cases.
+
+| Cases | Code | Highlighted | Group restriction | User resctriction | Is Visible Voucher page ? |
+|-------|------|-------------|-------------------|-------------------|---------------------------|
+| 1     | X    | Yes         | X                 | X                 | Yes                       |
```

Comment:
What is the 'X'?

Reply:
Yes and No, I could replace by Yes or No, or Yes/No

Reply:
It will be clearer, yes.

Reply:
+1 to replace the remaining "X", it's not very clear like this